src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -36,6 +45,73 @@ public String getKey() {
 
     @Override
     public String format(String value) {
+        Objects.requireNonNull(value);
+        // Handle case names in order lastname, firstname and separated by ","
+        // E.g., Ali Babar, M., Dingsøyr, T., Lago, P., van der Vliet, H.
+        if (!value.contains(" and ") && !value.contains("{") && !value.contains(";")) {
+            String[] valueParts = value.split(",");
+            // Delete spaces for correct case identification
+            for(int i=0; i < valueParts.length; i++) {
+                valueParts[i] = valueParts[i].trim();
+            }
+            // Looking for space between pre- and lastname
+            boolean spaceInAllParts = false;
+            for (int i=0; i<valueParts.length; i++) {
+                if (valueParts[i].contains(" ") ) {
+                    spaceInAllParts = true;
+                } else {
+                    spaceInAllParts = false;
+                    break;
+                }
+            }
+
+            // We hit the comma name separator case
+            // Usually the getAsLastFirstNamesWithAnd method would separate them if pre- and lastname are separated with "and"
+            // If not, we check if spaces separate pre- and lastname
+            if (spaceInAllParts) {
+                value = value.replaceAll(",", " and");
+            } else {
+                // Looking for name affixes to avoid
+                // partCount need to reduce by the count off avoiding terms
```

Comment:
The variable is now called "valuePartsCount".
"valuePartsCount holds the count of name parts without the avoided terms"